lib/catalog/determine_task_relevancy.rb

```diff
@@ -36,9 +42,9 @@ def delegate_task
     end
 
     def process_relevant_context
-      if @task.context&.has_key_path?(:service_instance)
+      if @task.output&.has_key?(:url)
         UpdateOrderItem.new(@task, @order_item).process
-      elsif @task.context&.has_key_path?(:applied_inventories)
+      elsif @task.output&.has_key_path?(:applied_inventories)
```

Comment:
@hsong-rh I think this elsif block is dead code we are no longer going to get inventories via Kafka

Reply:
removed